inst/templates/gitlab_ci/dot.gitlab-ci.yml

```diff
@@ -0,0 +1,36 @@
+variables:
+  # For caching renv library
+  RENV_PATHS_CACHE: ${CI_PROJECT_DIR}/cache
+  RENV_PATHS_LIBRARY: ${CI_PROJECT_DIR}/renv/library
+  # Cache .rhino
+  DOT_RHINO_PATH: ${CI_PROJECT_DIR}/.rhino
+  # Necessary for cypress to run.
+  # https://docs.cypress.io/guides/references/error-messages#A-Cached-Cypress-Binary-Could-not-be-found
+  # https://docs.cypress.io/guides/continuous-integration/introduction#Caching
+  CYPRESS_CACHE_FOLDER: ${CI_PROJECT_DIR}/.cache/Cypress
+
+
+default:
+  # Use deployed image with .gitlab-ci.Dockerfile
+  image: your/image:latest
```

Comment:
Users should use the image they built and pushed using the `.gitlab-ci.Dockerfile` template.